component/sections/call_to_action/variant_a.js

```diff
@@ -29,17 +29,19 @@ function VariantA({ logo, title, text, button }) {
             {button && (
               <a
                 className="inline-block py-2 px-6 bg-webriq-blue hover:bg-webriq-darkblue text-white font-bold leading-loose rounded-l-xl rounded-t-xl transition duration-200"
-                target={button?.pageAccess === "openLinkToNewTab" ? "_blank" : null}
-                rel={button?.pageAccess === "openLinkToNewTab" ? "noopener noreferrer" : null}
+                target={button?.linkTarget}
+                rel={
+                  button?.linkTarget === "_blank" ? "noopener noreferrer" : ""
```

Comment:
When condition returns false, it will set value to empty string vs null (nothing).